### PR TITLE
fix permission viewer

### PR DESCRIPTION
**Pull request by sadan4**

### CODE REVIEW

File: src/plugins/permissionsViewer/index.tsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ export default definePlugin({
         {
             find: ".VIEW_ALL_ROLES,",
             replacement: {
-                match: /\.collapseButton,.+?}\)}\),/,
+                match: /\.collapseButton,.+?null,/,
```
Comment (Nuckyz) [inline src/plugins/permissionsViewer/index.tsx:175]:
```suggestion
                match: /\.expandButton,.+?null,/,
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/plugins/permissionsViewer/index.tsx (hunk 1 above)
Comment (sadan4) [reply src/plugins/permissionsViewer/index.tsx:175]: i think this doesnt change the resulting patch, but idrc

Comment (Nuckyz) [reply src/plugins/permissionsViewer/index.tsx]: They swapped the order so it's more right

Comment (sadan4) [reply src/plugins/permissionsViewer/index.tsx]: ah

**Pull request merged**